app/views/layouts/_header.html.haml

```diff
@@ -32,6 +35,9 @@
   = render :partial => "layouts/spinner"
   = render :partial => "layouts/lightbox_panel"
 
+:javascript
+  miq_bootstrap('.navbar.navbar-pf-vertical');
```

Comment:
I wonder if we should group this notification related functionality in a separate ng module, something like `miq.notifications` perhaps? That way we can just instantiate that one here..